mobly/controllers/android_device.py

```diff
@@ -901,7 +901,7 @@ def root_adb(self):
     # So we need to wait for the device to come back before proceeding.
     self.adb.wait_for_device(timeout=DEFAULT_TIMEOUT_BOOT_COMPLETION_SECOND)
 
-  def load_snippet(self, name, package):
+  def load_snippet(self, name, package, instrument_options=None):
```

Comment:
is there any other params that might make sense to pass in here?

Bigger question is, should we add instrumentation options, or a config type?

Reply:
Good point. I think using a config type would be better. It's possible that other params will be needed in future.

Working on adding it.

Reply:
Well, to be fair, it took many years for us to need this one..
Maybe check what other things have to be passed in from cli and fit that into the config type as well?

Reply:
I took a look on it but not sure whether other params are needed. So I think we can add params when we really need them.